pkg/server/server.go

```diff
@@ -166,6 +169,20 @@ func (s *SSIServer) PresentationAPI(service svcframework.Service) (err error) {
 	s.Handle(http.MethodPut, handlerPath, pRouter.CreatePresentationDefinition)
 	s.Handle(http.MethodGet, path.Join(handlerPath, "/:id"), pRouter.GetPresentationDefinition)
 	s.Handle(http.MethodDelete, path.Join(handlerPath, "/:id"), pRouter.DeletePresentationDefinition)
+
+	return
+}
+func (s *SSIServer) SubmissionAPI(service svcframework.Service) (err error) {
```

Comment:
along with above - would like to see this a part of the PresentationAPI

Reply:
Done.